site/docs/pt/guide/getting-started.md

```diff
@@ -0,0 +1,222 @@
+# Primeiros Passos
+
+Crie seu primeiro Bot em minutos. ([Desça um pouco](#getting-started-on-deno) para um guia para um projeto Deno.)
+
+## Primeiros passos no Node.js
+
+> Esse guia assume que você tem o [Node.js](https://nodejs.org) instalado, e o `npm` deve vir com ele.
+> Se você não sabe o que são essas coisas, dê uma olhada na nossa [Introdução](./introduction)!
+
+Crie um novo projeto TypeScript e instale o pacote `grammy`.
+Para fazer isso, abra um terminal e digite:
+
+::: code-group
+
+```sh [npm]
+# Crie um novo diretório e entre nele.
+mkdir meu-bot
+cd meu-bot
+
+# Configure o TypeScript (ignore esse passo se você usa JavaScript).
+npm install -D typescript
+npx tsc --init
+
+# Instale o grammY.
+npm install grammy
+```
+
+```sh [Yarn]
+# Crie um novo diretório e entre nele.
+mkdir meu-bot
+cd meu-bot
+
+# Configure o TypeScript (ignore esse passo se você usa JavaScript).
+yarn add typescript -D
+npx tsc --init
+
+# Instale o grammY.
+yarn add grammy
+```
+
+```sh [pnpm]
+# Crie um novo diretório e entre nele.
+mkdir meu-bot
+cd meu-bot
+
+# Configure o TypeScript (ignore esse passo se você usa JavaScript).
+pnpm add -D typescript
+npx tsc --init
+
+# Instale o grammY.
+pnpm add grammy
+```
+
+:::
+
+Crie um novo arquivo de texto vazio chamado `bot.ts`, por exemplo.
+Sua estrutura de diretórios deve ficar mais ou menos assim:
+
+```asciiart:no-line-numbers
+.
+├── bot.ts
+├── node_modules/
+├── package.json
+├── package-lock.json
+└── tsconfig.json
+```
+
+Agora, é hora de abrir o Telegram para criar uma conta de bot e obter um token de bot para ele.
```

Comment:
```suggestion
Agora, é hora de abrir o Telegram para criar uma conta de bot e obter um token de bot para ela.
```
I think the personal pronoun here is referring to "conta de bot", so it should be in the femine form.

Reply:
I agree! I didn't like how this sounded initially too, but didn't really know what to do with it. Thanks!